docs/en/revocation-lists.rst

```diff
@@ -359,48 +381,63 @@ The *Credential status endpoint* MUST return a response with a *HTTP status code
       - It MUST contain the Status Attestation as a signed JWT. 
       - `[OAuth Status Attestation draft 01] <https://datatracker.ietf.org/doc/draft-demarco-status-attestations/01/>`_.
 
-If the Digital Credential could not be found by the Issuer, an HTTP Response with status code 404 (Not Found) MUST be returned. In all other cases the Issuer MUST return an HTTP Response Error using *application/json* as the content type, and including the following parameters:
-
-  - *error*. The error code.
-  - *error_description*. Text in human-readable form that offers more details to clarify the nature of the error encountered (for instance, changes in some attributes, reasons for revocation, other).
-
-Below a non-normative example of an HTTP Response with an error.
-
-.. code::
-
-  HTTP/1.1 400 Bad Request
-  Content-Type: application/json;charset=UTF-8
-
-  {
-    "error": "invalid_request"
-    "error_description": "The signature of credential_pop JWT is not valid"
-  }
-
-The following HTTP Status Codes and Error Codes MUST be supported:
+The following HTTP Status Codes MUST be supported:
 
 .. list-table:: 
     :widths: 20 20 60
     :header-rows: 1
 
     * - **Status Code**
-      - **Error Code**
+      - **Body**
       - **Description**
+    * - *201 Created*
+      - Status Attestation response
+      - The Status Attestation has been successfully created and it has been returned.  
     * - *400 Bad Request*
-      - *invalid_request*
-      - The request is not valid due to the lack or incorrectness of one or more parameters. (:rfc:`6749#section-5.2`).
+      - Error code and description
+      - The issuer cannot fulfill the request because of invalid parameters.
+    * - *404 Not Found*
+      - 
+      - The Digital Credential can not be found by the Issuer.
     * - *500 Internal Server Error*
-      - *server_error*
+      - 
```

Comment:
this is RFC6749

https://datatracker.ietf.org/doc/html/rfc6749#section-4.1.2.1